watertap/unit_models/anaerobic_digester.py

```diff
@@ -836,6 +1001,21 @@ def calculate_scaling_factors(self):
             & self.liquid_phase.properties_out.component_list
         )
 
+        iscale.set_scaling_factor(self.KH_co2, 1e2)
+        iscale.set_scaling_factor(self.KH_ch4, 1e2)
+        iscale.set_scaling_factor(self.KH_h2, 1e2)
+        iscale.set_scaling_factor(self.hydraulic_retention_time, 1e-6)
+        iscale.set_scaling_factor(self.volume_AD, 1e-2)
+        iscale.set_scaling_factor(self.volume_vapor, 1e-2)
+        iscale.set_scaling_factor(self.liquid_phase.rate_reaction_generation, 1e4)
+        iscale.set_scaling_factor(self.liquid_phase.mass_transfer_term, 1e2)
+        iscale.set_scaling_factor(self.liquid_phase.rate_reaction_extent, 1e4)
+        iscale.set_scaling_factor(self.liquid_phase.enthalpy_transfer, 1e0)
+        iscale.set_scaling_factor(self.liquid_phase.volume, 1e-2)
+        iscale.set_scaling_factor(self.electricity_consumption, 1e0)
+        for i, c in self.ad_performance_eqn.items():
+            iscale.constraint_scaling_transform(c, 1e2)
+
         # TODO: improve this later; for now, this resolved some scaling issues for modified adm1 test file
         if "S_IP" in self.config.liquid_property_package.component_list:
```

Comment:
we should revisit these at some point. Could be possible that these default factors render the scaled values small enough to satisfy constraint residual tolerances erroneously; i.e., solver reports optimal convergence but the real constraint is not satisfied within tolerance.

Not saying that this IS the case but that it could be a possibility that we should rule out.

Reply:
Sure, but just to provide some additional context, this particular line is from the old scaling routine. Rather than just remove the iscale functionality, I figured I'd leave it intact so that we can toggle between the old (iscale) and new (scaler objects) scaling routines when updating the BSM2 flowsheets